xds/src/main/java/io/grpc/xds/CdsLoadBalancer.java

```diff
@@ -74,38 +74,28 @@
   @VisibleForTesting
   CdsLoadBalancer(Helper helper, LoadBalancerRegistry lbRegistry,
       TlsContextManager tlsContextManager) {
-    this.helper = helper;
-    this.channelLogger = helper.getChannelLogger();
+    logger = XdsLogger.withLogId(InternalLogId.allocate("cds-lb", null));
```

Comment:
You can pass `helper.getAuthority()` to the 2nd arg of `allocate()`.

Reply:
Done. Same for EDS lb policy. But I doubt if this is useful as each LB policies has its internal resource resolution with xDS resource names. We don't have separate log prefixes for the inner components (i.e., `ClusterBalancer`, `ClusterEndpointsLoadBalancer`, etc) that do the real load balancing. Anyway, it would be fine for alpha as it will not have any cluster/cluster_service switches and LB policy instances in the hierarchy do not change.

Reply:
I agree `helper.getAuthority()` only helps temporarily. We may have better tracking of the component tree in the future.